maat-court-data-api/src/test/java/gov/uk/courtdata/builder/TestEntityDataBuilder.java

```diff
@@ -589,6 +589,36 @@ public static ContributionFilesEntity getContributionFilesEntity() {
                 .build();
     }
 
+    public static ContributionFilesEntity getPopulatedContributionFilesEntity(Integer fileId) {
```

Comment:
Since it is a primary key `Integer fileId` should probably be an int.
e.g.
`int fileId`

Reply:
@stone-dj I wanted people to be able to construct an instance with a `null` `id` - so could use as a test `Entity` with `JpaRepository.save`, etc., so that's why I'm allowing `null` argument. I could just have no parameter and return an instance with a `null` `id`, and required consumers to call `setId`, for example?